### PR TITLE
[atip] Update the method to get page url

### PR DESCRIPTION
**Pull request by chenxix**
Just change the method get_page_url() to get page url by test platform name.

### CODE REVIEW

**Comment by chenxix** [comment]
After the new param tizen_user is added, the original get_page_url() function cannot get the url correctly, just update it. and the result is OK.

File: tools/atip/atip/web/steps.py [hunk 1]
```diff
@@ -37,14 +37,18 @@ def get_page_url(context, text):
...
-        if str(context.app.app_config).upper().find('TIZEN') >= 0:
+        if test_platform == 'TIZEN':
             test_prefix = '%s://%s//' % (url_components.scheme,
                                          url_components.netloc)
-        elif str(context.app.app_config).upper().find('ANDROID') >= 0:
+        elif test_platform == 'ANDROID':
             if url_components.scheme == 'http':
                 test_prefix = '%s://%s/' % (url_components.scheme,
                                             url_components.netloc)
+        else:
```
Comment (Honry) [inline tools/atip/atip/web/steps.py]: Testplatform is not a word, please separate it.

**Comment by Honry** [comment]
LGTM 
@cicili please help review.

**Comment by cicili** [comment]
LGTM

**Pull request merged**